src/commands/schedule.ts

```diff
@@ -74,7 +74,7 @@ const schedule = async (options: CommandExecuteParameters) => {
 	const cronString = dateToCron(new Date((`${date} ${time}`).trim()))
 	// create the cronjob
 	const prisma = new PrismaClient()
-	const startMessage = await channel.send(`Game time: <t:${Math.floor(dateString.getTime() / 1000)}:F>`)
+	const startMessage = await channel.send(`Game time: <t:${Math.floor(dateString.getTime() / 1000)}:F> || <t:${Math.floor(dateString.getTime() / 1000)}:R>`)
```

Comment:
This adds relative time. See Github Gist [Here](https://gist.github.com/LeviSnoot/d9147767abeef2f770e9ddcd91eb85aa#discord-timestamps).